bundles/org.openhab.binding.elroconnects/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,286 @@
+		<channels>
+			<channel id="entryAlarm" typeId="entryalarm"/>
+			<channel id="entry" typeId="entry"/>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="lowBattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="motionsensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Motion Sensor</label>
+		<description>ELRO Connects motion sensor</description>
+		<channels>
+			<channel id="motionAlarm" typeId="motionalarm"/>
+			<channel id="motion" typeId="system.motion"/>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="lowBattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="temperaturesensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Temperature Sensor</label>
+		<description>ELRO Connects motion sensor</description>
+		<channels>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="humidity" typeId="system.atmospheric-humidity"/>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="lowBattery" typeId="system.low-battery"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<thing-type id="powersocket">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="connector"/>
+		</supported-bridge-type-refs>
+		<label>Power Socket</label>
+		<description>ELRO Connects power socket</description>
+		<channels>
+			<channel id="powerState" typeId="system.power"/>
+		</channels>
+		<representation-property>deviceId</representation-property>
+		<config-description>
+			<parameter name="deviceId" type="integer" required="true">
+				<label>Device ID</label>
+				<description>ID of the ELRO Connects Device.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="scene">
+		<item-type>String</item-type>
+		<label>Scene</label>
+		<description>Scene selection from scenes configured in the Elro Connects app, enables configuring alarm modes</description>
+	</channel-type>
+
+	<channel-type id="smokealarm">
```

Comment:
```suggestion
	<channel-type id="smokeAlarm">
```
I'm pretty sure you changed to using lowerCamelCase in the readme, but it appears they are not changed here. Please look at all channels as I wont raise a comment on each one.

Reply:
@Skinah I changed the channel id's to lowerCamelCase, but not the channel-type id's. I believe it is the channel id's that are visible to the user when configuring. I also use system channel type id's and they have yet another convention (system.xxx-xxx). There is a mistake in the README by the way I will correct. It should be Channel ID, not Channel Type ID.
Do you want me to change the channel type ID's to lowerCamelCase as well? And what about the thing type ID's?

Reply:
So long as the channels that the user sees and has to do textual config with, that is all that matters.